zk_test.go

```diff
@@ -97,6 +97,84 @@ func TestCreate(t *testing.T) {
 	}
 }
 
+func TestCreateTTL(t *testing.T) {
+	ts, err := StartTestCluster(t, 1, nil, logWriter{t: t, p: "[ZKERR] "})
+	if err != nil {
+		t.Fatal(err)
+	}
+	defer ts.Stop()
+	zk, _, err := ts.ConnectAll()
+	if err != nil {
+		t.Fatalf("Connect returned error: %+v", err)
+	}
+	defer zk.Close()
+
+	path := "/gozk-test-ttl"
+
+	if err := zk.Delete(path, -1); err != nil && err != ErrNoNode {
+		t.Fatalf("Delete returned error: %+v", err)
+	}
+	if p, err := zk.CreateTTL(path, []byte{1, 2, 3, 4}, FlagWithTTL, WorldACL(PermAll), 60*1000); err != nil {
```

Comment:
`FlagWithTTL | FlagSequence` to verify the combination working

Reply:
Just as 24's answer